examples/UAVCAN-BMS-Subscribe/UAVCAN-BMS-Subscribe.ino

```diff
@@ -0,0 +1,207 @@
+/*
+ *  This example file shows how to read messages from a UAVCAN v1 enable BMS and display
+ *   the information in console output.
+ *
+ *  Hardware:
+     - UAVCAN v1 enabled BMS (eg: https://rotoye.com/batmon/ and
+        wiring guide: https://rotoye.com/batmon-arduino-can-bus-wiring-guide-and-uavcan-v1/)
+     - A (multicell) battery or battery simulator
+ *   - Arduino MKR family board, e.g. MKR Zero
+ *   - Arduino MKR CAN shield
+ *
+ *  Copyright (c) 2020, SkyMul Inc.
```

Comment:
License must be changed to MIT and copyright to Alexander Entinger, LXRobotics. Better still, just drop it and replace it with a authors notice (Authors: Kyle Pawlowski, Eohan George eohan@rotoye.com). You are not protecting any valuable copyright, but are polluting this repository with a license != MIT.

Reply:
why should it be copyright to LXRobotics?

Reply:
It doesn't need to be copyrighted to LXR, but it can't be GPL for sure. If everyone who submits some code to this library uses the license he desires, than this will result in a big ball of mess of different licenses. Your example contains no code that would warrant any copyrighting with GPL, besides it's a ~75% copy of existing example code written by me, therefore it is already in violation of code I published under the MIT license ... Put yourself as author and link to your product, but don't go GPL on me :wink:

Reply:
fair enough. will make few changes.

Reply:
When can I expect that update?

Reply:
Little tied up. Should be in before Tuesday

Reply:
Done. good ?

Reply:
Updated. Is that good ?